packages/engine/package.json

```diff
@@ -50,9 +50,9 @@
     "detect-gpu": "^5.0.38",
     "draco3dgltf": "^1.5.6",
     "fflate": "0.7.4",
-    "hls.js": "^1.3.5",
```

Comment:
I also updated hls because they said it would lower bundle size but it didn't get that much lower :(